src/app/routes/enrolment/fields/fields.component.ts

```diff
@@ -9,5 +16,19 @@ import { KeyValue } from '@angular/common';
 })
 export class FieldsComponent {
   @Input() title: string;
+  @Input() isIssuerRequestView = false;
   @Input() fieldsList: KeyValue<string, string | number>[];
+  @Output() fieldToCopy = new EventEmitter<KeyValue<string, string | number>>();
+  @Input() issuerFields: IFieldDefinition[] | undefined;
+  hasValidValue(val: string | number) {
+    return val && val !== 'null';
```

Comment:
In some of the old mining forms, the null values were coming back as "null", not sure if this happens other times when parsing credential fields or if it was just bad code in Gp4btc at one point.